third_party/kiwi/kiwi/be_read.h

```diff
@@ -17,6 +17,8 @@ struct kiwi_be_startup {
 	kiwi_var_t user;
 	kiwi_var_t database;
 	kiwi_var_t replication;
+	kiwi_var_t lsu_username;
```

Comment:
Storing such settings in client->rule instead of client->startup would be better i think.

Reply:
I'm tried to store this settings in client->rule at [debug-branch](e7085dd45d59d91028965b2954fb6d1753422be2), but this variable has overridden somewhere near [od_fronted_startup](https://github.com/yandex/odyssey/blob/f9a1ec21bc908ce4814b3ca2cfc7f1028204e466/sources/frontend.c#L114) by another client and race come again...